docs/api/README.md

```diff
@@ -0,0 +1,212 @@
+ - `param=201530` *(A single epiweek)*
+ - `param=201401,201501,201601` *(Several epiweeks)*
+ - `param=200501-200552` *(A range of epiweeks)*
+ - `param=201440,201501-201510` *(Several epiweeks, including a range)*
+ - `param=20070101-20071231` *(A range of dates)*
+
+## Universal Parameters
+
+The only universally required parameter is `source`, which must be one of the supported source names listed below, e.g., `fluview`.
+
+## Source-Specific Parameters
+
+The parameters available for each source are documented in each linked source-specific API page.
+
+<!-- TODO: check existing descriptions -->
+
+### COVID-19 Data
+
+| Source | Name | Description | Restricted? |
+| --- | --- | --- | --- |
+| [`covidcast`](covidcast.md) | COVIDCast | Delphi's COVID-19 surveillance streams. | no |
+| [`covidcast_meta`](covidcast_meta.md) | COVIDCast Metadata | Metadata for Delphi's COVID-19 surveillance streams. | no |
+
+### Influenza Data
+
+| Source | Name | Description | Restricted? |
+| --- | --- | --- | --- |
+| [`afhsb`](ahfsb.md) | AFHSB | ... <!-- TODO --> | yes |
+| [`cdc`](cdc.md) | CDC Page Hits | ... <!-- TODO --> | yes |
+| [`delphi`](delphi.md) | Delphi's Forecast | ... <!-- TODO --> | no |
+| [`ecdc_ili`](ecdc_ili.md) | ECDC ILI | ECDC ILI data from the ECDC website. | no |
+| [`flusurv`](flusurv.md) | FluSurv | FluSurv-NET data (flu hospitaliation rates) from CDC. | no |
+| [`fluview`](fluview.md) | FluView | Influenza-like illness (ILI) from U.S. Outpatient Influenza-like Illness Surveillance Network (ILINet). | no |
+| [`fluview_clinical`](fluview_clinical.md) | FluView Clinical | ... <!-- TODO --> | no |
+| [`gft`](gft.md) | Google Flu Trends | Estimate of influenza activity based on volume of certain search queries. Google has discontinued Flu Trends, and this is now a static data source. | no |
+| [`ght`](ght.md) | Google Health Trends | Estimate of influenza activity based on volume of certain search queries. | yes |
+| [`kcdc_ili`](kcdc_ili.md) | KCDC ILI | KCDC ILI data from KCDC website. | no |
+| [`meta`](meta.md) | API Metadata | Metadata for `fluview`, `twitter`, `wiki`, and `delphi`. | no |
+| [`meta_afhsb`](meta_afhsb.md) | AFHSB Metadata | ... <!-- TODO --> | yes |
+| [`nidss_flu`](nidss_flu.md) | NIDSS Flu | Outpatient ILI from Taiwan's National Infectious Disease Statistics System (NIDSS). | no |
+| [`nowcast`](nowcast.md) | ILI Nearby | A nowcast of U.S. national, regional, and state-level (weighted) percent ILI, available seven days (regionally) or five days (state-level) before the first ILINet report for the corresponding week. | no |
+| [`quidel`](quidel.md) | Quidel | Data provided by Quidel Corp., which contains flu lab test results. | yes |
+| [`sensors`](sensors.md) | Delphi's Digital Surveillance Sensors | ... <!-- TODO --> | no |
+| [`twitter`](twitter.md) | Twitter Stream | Estimate of influenza activity based on analysis of language used in tweets. | yes |
```

Comment:
we should prominently acknowledge http://HealthTweets.org/

Reply:
added link to description.